scripts/build_package.sh

```diff
@@ -15,15 +14,15 @@ python setup.py bdist_wheel --universal
 
 # Make sure we have a correct version
 if [[ -n "$TRAVIS_TAG" ]]; then
-    pip uninstall -y dvc
-    if which dvc; then
-        echo "ERROR: dvc command still exists! Unable to verify dvc version."
-	exit 1
-    fi
-    pip install dist/dvc-*.whl
-    if [[ "$(dvc --version)" != "$TRAVIS_TAG" ]]; then
-        echo "ERROR: 'dvc --version'$(dvc -V) doesn't match '$TRAVIS_TAG'"
-        exit 1
-    fi
-    pip uninstall -y dvc
+  pip uninstall -y dvc
+  if which dvc; then
+    echo "ERROR: dvc command still exists! Unable to verify dvc version." >&2
+    exit 1
+  fi
+  pip install dist/dvc-*.whl
+  if [[ "$(dvc --version)" != "$TRAVIS_TAG" ]]; then
+    echo "ERROR: 'dvc --version'$(dvc -V) doesn't match '$TRAVIS_TAG'" >&2
+    exit 1
+  fi
+  pip uninstall -y dvc
 fi
```

Comment:
Does't look related to snap tests, let's not do this in this PR. Consider creating a separate one. (Btw we even have a ticket somwhere to run shell formatter on all our shell scripts to handle this mess 🙂 )

Reply:
opened #2969